### DISCUSSION TITLE
How to ignore semver tags for a go repo

### DISCUSSION THREAD

**Discussion by jmcrawford45** [renovatebot/renovate#28793] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

Hi,
I have a use case where a polyglot repo pushes semver tags for its other languages, but these versions won't resolve in go due to https://go.dev/ref/mod#glos-module-subdirectory. So, I'd like to force the versioning strategy to only use digest versions, which do resolve correctly. I looked through https://docs.renovatebot.com/modules/versioning/, but I don't see any relevant values for this behavior. Is this something that's supported? Ideally, I'd like something like

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "packageRules": [
    {
      "matchPackagePatterns": [
        ".*/my/repo/submodule-go$",
      ],
      "datasources": ["go"],
      "versioning": "digest",
      "enabled": true,
      "stabilityDays": 30
    }
  ]
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Are you using a Go proxy or is that an option? Or does the go proxy also produce results you don't agree with?
